scbd/absch.cbd.int
# chm-414  sixth National Report  view page

### General description
migreate 6th national report view page.

### Designs
same as old chm project and absch project page.

### Testing instructions
search -> 6th nation report

please note this pull request  should approve after  "update process assessment page" (https://github.com/scbd/absch.cbd.int/pull/1164) , becaues it need process assessment page to display correctly.

nested  national target page:
<img width="915" alt="image" src="https://github.com/scbd/absch.cbd.int/assets/65098066/8976b83c-6d01-42f5-a7b9-ca547b557957">

nested progress assessment page:
<img width="892" alt="image" src="https://github.com/scbd/absch.cbd.int/assets/65098066/41df31f4-67a4-40d9-a456-d2e431ff5860">

footer reference
<img width="905" alt="image" src="https://github.com/scbd/absch.cbd.int/assets/65098066/bf1a6d36-8d41-425f-9691-2dacb4b5065a">
